python/simulation.py

```diff
@@ -4278,6 +4278,25 @@ def change_k_point(self, k):
                     self.fields.use_bloch(
                         py_v3_to_vec(self.dimensions, self.k_point, self.is_cylindrical)
                     )
+    
+    def change_m_number(self, m):
```

Comment:
@stevengj, I tried creating a function that updates an existing simulation's `m` number. Even with this new function, however, the `m` number does not seem to change. In other words, the resulting adjoint fields are the same regardless if this function is called or not.

Any ideas on the proper way to update `m`?

Reply:
Whoops forgot to update the `fields_chunk` too. Should be good now.